config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins "example.com"
+#
+#     resource "*",
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
+# end
```

Comment:
[OPTIONAL] - I recommend setting the cors file to avoid problems when connecting the API to the Front-End :slightly_smiling_face:

Reply:
OK. Thanks, @leopiresgoss. We will handle it when creating API endpoints.